tests/app/test_surveys.py

```diff
@@ -70,6 +73,42 @@ def test_surveys_todo(self, mock_request):
         self.assertTrue('Business Register and Employment Survey'.encode() in response.data)
         self.assertTrue('RUNAME1_COMPANY4 RUNNAME2_COMPANY4'.encode() in response.data)
 
+    @requests_mock.mock()
```

Comment:
What happens if the '/surveys/todo` endpoint returns an error?

Reply:
This should be covered in another test as these tests are only checking for the trading as and not if the page can be displayed.